agent/command/git.go

```diff
@@ -533,6 +542,25 @@ func (c *gitFetchProject) executeLoop(ctx context.Context,
 
 	//Apply patches if this is a patch and we haven't already gotten the changes from a PR
 	if evergreen.IsPatchRequester(conf.Task.Requester) && conf.GithubPatchData.PRNumber == 0 {
+		for _, patchId := range c.AdditionalPatches {
+			if bson.IsObjectIdHex(patchId) {
+				p, err = comm.GetTaskPatch(ctx, td)
```

Comment:
Does this actually use patchId? It seems like it'd be getting the patch for the task listed in td

Reply:
🤦 I knew this was too easy. I added support for this in the route that returns the patch